src/components/Tooltip/Tooltip.module.scss

```diff
@@ -0,0 +1,35 @@
+.tooltip {
+    display: inline-block;
+    margin: 0;
+    position: relative;
+    border-bottom: 1px dotted gray;
+}
+
+.hint {
+    display: inline-block;
+    visibility: hidden;
+    z-index: 1;
+    position: absolute;
+    bottom: 120%;
+    right: -15%;
+    border: 2px dashed black;
+    border-radius: 5px;
+    background: white;
+    min-width: 15vw;
+    max-width: 40vw;
+    padding: 10px 10px 10px 10px;
+}
+
+.tooltip .hint::after {
+    content: " ";
+    position: absolute;
+    top: 100%;
+    right: 5%;
+    border-width: 10px;
+    border-style: dashed;
+    border-color: black transparent transparent transparent;
```

Comment:
Tenker med en gang på spacing-tokens når jeg ser pixel verdier som dette, men etter en nærmere inspeksjon ser det ikke ut som at vi har [Infima sine spacing tokens](https://infima.dev/docs/layout/spacing) i vår liste over CSS variabler.

Det kan eventuelt komme inn som et separat issue (om det kan gjøres via pakke-oppdatering, eller om vi gjør det manuelt), jeg syns det ser bra ut for nå!